params/config.go

```diff
@@ -62,13 +62,12 @@ var CheckpointOracles = map[common.Hash]*CheckpointOracleConfig{
 }
 
 var (
-	bigBondedStakeNewton = big.NewInt(10_000)
-	bigPrecisionFactor   = big.NewInt(1_000_000_000_000_000_000)
-	bigBondedStake       = big.NewInt(0).Mul(bigBondedStakeNewton, bigPrecisionFactor)
+	NtnPrecision = big.NewInt(1_000_000_000_000_000_000)
+	Ntn1         = new(big.Int).Mul(big.NewInt(1), NtnPrecision)
+	Ntn10000     = new(big.Int).Mul(big.NewInt(10_000), NtnPrecision)
 
-	// PiccaddillyChainConfig todo: ask Raj to generate validator key for validators in the PiccaddillyChainConfig
-	// PiccaddillyChainConfig contains the chain parameters to run a node on the Piccaddilly test network.
-	PiccaddillyChainConfig = &ChainConfig{
+	// PiccadillyChainConfig contains the chain parameters to run a node on the Piccaddilly test network.
+	PiccadillyChainConfig = &ChainConfig{
 		ChainID:                 big.NewInt(65_100_001),
```

Comment:
```suggestion
		ChainID:                 big.NewInt(65_100_0012),
```

Per APR25, the chain and network ID is `65100002`

Reply:
@yazzaoui - note also per https://github.com/ethereum-lists/chains/pull/4346 for Bakerloo Testnet it is changed to:

```
"chainId": 65010002,
"networkId": 65010002,
```